cua-server/src/index.ts

```diff
@@ -1,6 +1,11 @@
+import dotenv from "dotenv";
 import http from "http";
 import { Server as SocketIOServer, Socket } from "socket.io";
 
+
+// Load environment variables first
+dotenv.config();
```

Comment:
By default, dotenv.config() loads .env from the current working directory. To load environment-specific files (e.g., .env.development, .env.production), you must explicitly specify the file path using the path option.

```suggestion
dotenv.config({ path: '.env.development' });
```

https://github.com/motdotla/dotenv?tab=readme-ov-file#options

Reply:
try this code:

```javascript
import { loadEnvFile } from 'process';

loadEnvFile('./.env');
```